src/main/target/NAZE/target.h

```diff
@@ -193,5 +194,5 @@
 #define USED_TIMERS         (TIM_N(1) | TIM_N(2) | TIM_N(3) | TIM_N(4))
 
 #define TIMER_APB1_PERIPHERALS (RCC_APB1Periph_TIM2 | RCC_APB1Periph_TIM3 | RCC_APB1Periph_TIM4)
-#define TIMER_APB2_PERIPHERALS (RCC_APB2Periph_TIM1 | RCC_APB2Periph_GPIOA | RCC_APB2Periph_GPIOB)
+#define TIMER_APB2_PERIPHERALS (RCC_APB2Periph_TIM1 | BEEPER_GPIO_CLK_PERIPHERAL | RCC_APB2Periph_GPIOB)
```

Comment:
This reminds me, these are not needed, as they can be inferred from the Pins, and the Timers in the timerHardware array.